internal/engine/image_build_and_deployer.go

```diff
@@ -137,7 +137,7 @@ func (ibd *ImageBuildAndDeployer) build(ctx context.Context, manifest model.Mani
 
 		df := ibd.baseDockerfile(manifest, cacheRef)
 		steps := manifest.Steps
-		ref, err := ibd.b.BuildImageFromScratch(ctx, ps, name, df, manifest.Mounts, ignore.CreateBuildContextFilter(manifest), steps, manifest.Entrypoint)
+		ref, err := ibd.b.BuildImageFromScratch(ctx, ps, name, df, manifest.Mounts, ignore.CreateBuildContextFilter(manifest), steps, manifest.Entrypoint, manifest.StaticBuildArgs)
```

Comment:
i'm not sure what the intended semantics are here. like, is this if you want to have build args in Dockerfile.base?

maybe add a test for this case?

Reply:
Yeah good point, I did name this _static_ build args. I'll remove this param.